ibm_catalog.json

```diff
@@ -39,7 +39,7 @@
         },
         {
           "title": "KMS encryption",
-          "description": "Encrypt your data using the default encryption model or use IBM Key Protect or IBM Hyper Protect Crypto Services for full control over the data encryption keys used. [Learn more](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-encryption)."
+          "description": "Optionally encrypt your data using the default encryption model or use IBM Key Protect or IBM Hyper Protect Crypto Services for full control over the data encryption keys used. [Learn more](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-encryption)."
```

Comment:
Please rewrite as:
You can optionally choose to encrypt your data using the default encryption keys, or opt for IBM Key Protect or IBM Hyper Protect Crypto Services to have complete control over the encryption keys. Learn more.